Neos.Utility.ObjectHandling/Classes/TypeHandling.php

```diff
@@ -34,7 +34,7 @@ abstract class TypeHandling
     /**
      * @var array
      */
-    protected static $collectionTypes = ['array', \ArrayObject::class, \SplObjectStorage::class, Collection::class];
+    protected static $collectionTypes = ['array', \ArrayObject::class, \SplObjectStorage::class, Collection::class, \IteratorAggregate::class];
```

Comment:
If we change this we should not add the specific `IteratorAggregate` interface but the `Traversable` interface that is implemented by all the other classes already:
```suggestion
    protected static $collectionTypes = ['array', \Traversable::class];
```

I'm not 100% sure about possible side effects of changing this here.
A more b/c alternative might be to change
```php
} elseif ((is_object($value) && !TypeHandling::isCollectionType(get_class($value))) && !is_array($value)) {
```
to
```php
} elseif (!is_iterable($value)) {
```
in the [CollectionValidator](https://github.com/neos/flow-development-collection/blob/8b69724089ed4a55a37b2fe383c7f510f6272a5d/Neos.Flow/Classes/Validation/Validator/CollectionValidator.php#L55)

Reply:
The changes in TypeHandling is only untill this PR is pushed 
https://github.com/neos/flow-development-collection/pull/2202

Reply:
@bwaidelich see https://github.com/neos/flow-development-collection/pull/2202#discussion_r515609751 - had the exact same idea/concern :) Changing the `CollectionValidator` to `is_iterable` might be a sane alternative, if we are unsure about how many places uses `isCollection()` to implicitly also check for `Countable` and/or `ArrayAccess`